src/annotator/events.ts

```diff
@@ -0,0 +1,16 @@
+import type { SidebarLayout } from '../types/annotator';
+
+type LayoutChangeEventDetail = {
+  sideBySideActive: boolean;
+  sidebarLayout: SidebarLayout;
+};
+
+export class LayoutChangeEvent extends CustomEvent<LayoutChangeEventDetail> {
```

Comment:
I had a suggestion to add a common base class for all Hypothesis events. The initial definition could just be an empty extension of `CustomEvent`. This would allow doing things like `instanceof` checks, although the user could also check for the `hypothesis:` prefix on the type.

Reply:
I thought you were suggesting this as in a general improvement for later. If that was not the intention, I would suggest to actually do it that way and tackle it separately.

It would be good to put some thought on checking what common requirements should go in that base class, and consolidating our currently two existing events.

Reply:
> I thought you were suggesting this as in a general improvement for later.

Yeah, I was a bit vague about that. I don't think it is essential to make that change as part of this PR, I would suggest to do it "soon".